examples/svg-components/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "svg-components",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "beta"
+  },
+  "devDependencies": {
+    "babel-plugin-inline-react-svg": "^0.2.0"
```

Comment:
hmm, but maybe inlining is not optimal in case a svg was imported from multiple files. We can use https://github.com/boopathi/react-svg-loader instead ?